src/getter.jl

```diff
@@ -0,0 +1,443 @@
+
+!!! warning
+
+    This function differs from the `TLorentzVector::P()` function in the `ROOT` library.
+"""
+spatial_magnitude
+
+"""
+    polar_angle(lv)
+
+Return the polar angle (``\\theta``) of the Lorentz-vector-like `lv`.
+
+# Example
+
+If the Lorentz-vector-like is a four-vector ``(x, y, z, t)``, this is equivalent to ``\\arccos(z / \\sqrt{x^2 + y^2 + z^2})``.
+
+!!! note
+
+    If the Lorentz-vector-like is zero, the function returns zero for the angle.
+
+# See Also
+- [`LorentzVectorBase.cos_theta`](@ref): for a faster version of the cosine of the polar angle.
+"""
+polar_angle
+
+"""
+    cos_theta(lv)
+
+Return the cosine of the polar angle (``\\theta``) of the Lorentz-vector-like `lv`.
+
+Depending on the coordinate system, this might be an equivalent but faster version of `cos(polar_angle(lv))`.
+
+# Example
+
+If ``(x, y, z, t)`` is a four-vector, this is equivalent to ``z / \\sqrt{x^2 + y^2 + z^2}``.
+
+# See Also
+- [`LorentzVectorBase.polar_angle`](@ref): for the polar angle itself.
+"""
+cos_theta
+
+"""
+    phi(lv)
+
+Return the azimuthal angle (``\\phi``) of the Lorentz-vector-like `lv`.
+
+The azimuthal angle is the angle between the x-axis and the projection of the Lorentz-vector-like on the x-y plane.
+
+# Example
+    If ``(x, y, z, t)`` is a four-vector, this is equivalent to ``atan(y / x)``.
+
+!!! note
+
+    The azimuthal angle is defined with respect to the z-axis.
+
+# See Also
+- [`LorentzVectorBase.sin_phi`](@ref): for a faster version of the sine of the azimuthal angle.
+- [`LorentzVectorBase.cos_phi`](@ref): for a faster version of the cosine of the azimuthal angle.
+"""
+phi
+
+"""
+    cos_phi(lv)
+
+Return the cosine of the azimuthal angle (``\\phi``) of the Lorentz-vector-like `lv`.
+
+Depending on the coordinate system, this might be an equivalent but faster version of `cos(phi(lv))`.
+
+# Example
+If ``(x, y, z, t)`` is a four-momentum, this is equivalent to ``x / \\sqrt{x^2 + y^2}``.
+
+# See Also
+- [`LorentzVectorBase.phi`](@ref): for the azimuthal angle itself.
+- [`LorentzVectorBase.sin_phi`](@ref): for a faster version of the sine of the azimuthal angle.
+"""
+cos_phi
+
+"""
+    sin_phi(lv)
+
+Return the sine of the azimuthal angle (``\\phi``) of the Lorentz-vector-like `lv`.
+
+Depending on the coordinate system, this might be an equivalent but faster version of `sin(phi(lv))`.
+
+# Example
+If ``(x, y, z, t)`` is a four-momentum, this is equivalent to ``y / \\sqrt{x^2 + y^2}``.
+
+# See Also
+- [`LorentzVectorBase.phi`](@ref): for the azimuthal angle itself.
+- [`LorentzVectorBase.cos_phi`](@ref): for a faster version of the cosine of the azimuthal angle.
+"""
+sin_phi
+
+### Masses
+"""
+    mass2(lv)
+
+Return the squared invariant mass of the Lorentz-vector-like `lv`, computed as the Minkowski inner product of the four-vector components with itself.
+
+
+# Example
+For a four-momentum `(px, py, pz, E)`, this function returns `E² - (px² + py² + pz²)`.
```

Comment:
Should probably have these the same way as they are above, with proper latex math environments
```suggestion
For a four-momentum ``(p_x, p_y, p_z, E)``, this function returns ``E^2 - (p_x^2 + p_y^2 + p_z^2)``.
```
And similarly below.